src/loki/search.cc

```diff
@@ -25,7 +25,7 @@ constexpr float NODE_SNAP = 5.f;
 //during side of street computations we figured you're on the street if you are less than
 //5 meters (16) feet from the centerline. this is actually pretty large (with accurate shape
 //data for the roads it might want half that) but its better to assume on street than not
-constexpr float SIDE_OF_STREET_SNAP = 5.f;
+constexpr float SIDE_OF_STREET_SNAP = 25.f;
```

Comment:
since it uses the square distance now i changed this to the square, wasnt worth a `sqrt`